### PR TITLE
add github action

### PR DESCRIPTION
**Pull request by PoulainMaxime**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Clean dependencies
+author: Poulinho
+description: 'An easy GA to clean your dependencies before generate artifact'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  path:
+    description: 'Add path to your vuejs project'
```
Comment (Quetzacoalt91) [inline action.yml]:
```suggestion
    description: 'Additional files & folders to remove'
```

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Clean dependencies
+author: Poulinho
+description: 'An easy GA to clean your dependencies before generate artifact'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  path:
```
Comment (Quetzacoalt91) [inline action.yml]:
```suggestion
  files:
```

Comment (zalexki) [reply action.yml]: IMO path make more sens because we are doing `rm -rf /the/path` and not giving the possibility to give multiple files path

Comment (PoulainMaxime) [reply action.yml]: I'm thinking the same thing as @zalexki putting in input `files` could confuse the user.

Comment (Quetzacoalt91) [reply action.yml]: My point is to allow several files to be deleted actually, as `rm` accepts several arguments. I chose `files` as folders are a special type of file in a filesystem, but `paths` would be valid too.

Comment (PoulainMaxime) [reply action.yml]: All right, so can we just say this discussion can be resolved?

Comment (Quetzacoalt91) [reply action.yml]: I would write this argument in plural form at least.

Comment (PoulainMaxime) [reply action.yml]: I just made the change.

**Comment by Quetzacoalt91** [comment]
Thank you @PoulainMaxime

**Pull request merged**